src/components/calculators/Payslip.vue

```diff
@@ -3,67 +3,120 @@ Payslip calculator.
 -->
 <template>
     <div class="practiceweb-calculator">
-        <form @submit.prevent="submitCalculation">
-            <fieldset>
-            <div>
-                <p>
-                <label>Gross Pay: <input name="grossPay" v-validate="'required|decimal:2'" type="number" min="1" step="0.01" v-model.number="input.grossPay"></label>
-                <span v-show="errors.has('grossPay')"> {{ errors.first('grossPay') }}</span>
-                </p>
-            </div>
-            <div>
-            <label>Received
-                <label>Annual <input type="radio" v-model="input.period" value="annual"/></label>
-                <label>Monthly <input type="radio" v-model="input.period" value="month"/></label>
-                <label>Weekly <input type="radio" v-model="input.period" value="week"/></label>
-            </label>
-            </div>
+        <div class="practiceweb-calculator-header--width">
+            <h3 class="practiceweb-calculator-header">Payslip Calculator</h3>
+        </div>
+        <form class="input input--width" @submit.prevent="submitCalculation">
+            <fieldset class="input-fields">
+                <p class="input-fields__title">Gross Pay *</p>
+                <div class="form-group form-group--gross flex-wrap">
+                    <span class="input-field--specialCharacter input-field--specialCharacter--pound">&#163;</span>
+                    <input name="grossPay" v-validate="'required|decimal:2'" type="number" min="1" step="0.01" v-model.number="input.grossPay" class="input-field input-field--number border-left" placeholder="Enter your gross pay here"/>
+                    <label class="input-field--label flex-grow border-left" :class="{active: input.period === 'Annual'}"><input type="radio" v-model="input.period" value="Annual" class="input-field input-field--radio input-field--radio--hidden" :checked="checked"/>Annual</label>
```

Comment:
Line lengths are very long and not easy to scan to see what elements are included. You can split line lengths up to become more readable for others...

`<label class="input-field--label flex-grow border-left" :class="{active: input.period === 'Annual'}"><input type="radio" v-model="input.period" value="Annual" class="input-field input-field--radio input-field--radio--hidden" :checked="checked"/>Annual</label>`

Becomes...

```
<label class="input-field--label flex-grow border-left" 
       :class="{active: input.period === 'Annual'}">
  <input type="radio" 
         v-model="input.period"
         value="Annual"
         class="input-field input-field--radio input-field--radio--hidden" 
         :checked="checked"/>Annual
</label>
```

Reply:
Finished some structuring to make the code easier to read.